diff --git a/dist/react-error-boundary.js b/dist/react-error-boundary.js
index 50707fb1701cda610d9581bb8325047335c3e406..9cd93784e0725759f3c0f93d5214e4ea9bc863ba 100644
--- a/dist/react-error-boundary.js
+++ b/dist/react-error-boundary.js
@@ -1,3 +1,4 @@
+"use client";
 var $8zHUo$react = require("react");
 
 function $parcel$exportWildcard(dest, source) {
@@ -158,7 +159,6 @@ function $9e88dd86e0bb2944$export$f0c7a449e0cfaec7(component, errorBoundaryProps
 var $faefaad95e5fcca0$exports = {};
 
 
-"use client";
 $parcel$exportWildcard(module.exports, $6d6d4999e62b3ee0$exports);
 $parcel$exportWildcard(module.exports, $4a61716688322eb0$exports);
 $parcel$exportWildcard(module.exports, $3c4937b727a6fcfb$exports);